### PR TITLE
accessiblity for nav buttons

### PR DESCRIPTION
**Pull request by sorcerix**
added labels to buttons, will be part of a much larger accessibility sweep in another ticket,  and adjusted the color contrast between the inside and outside of the nav buttons

### CODE REVIEW

File: components/TourBar/TourNavButton.tsx [hunk 1]
```diff
@@ -1,19 +1,27 @@
...
+import ArrowLeftIcon from '@material-ui/icons/ArrowLeft';
+import ArrowRightIcon from '@material-ui/icons/ArrowRight';
 
 import { MobileScreenSize } from '../constants';
 
 const NavButton = UITheme.button`
 
     display: inline-block;
+
+    display: inline-block;
```
Comment (SuperStingray) [inline components/TourBar/TourNavButton.tsx]: Looks like this is duplicated.

Comment (sorcerix) [reply components/TourBar/TourNavButton.tsx]: thanks, fixing!

File: package-lock.json [hunk 2]
```diff
@@ -3931,6 +3931,12 @@
       "resolved": "https://registry.npmjs.org/arr-union/-/arr-union-3.1.0.tgz",
```
Comment (SuperStingray) [inline package-lock.json:3931]: Were all the changes in this file intended?

**Comment by odbol** [comment]
Thanks for doing this! Looks great!

**Review by odbol — APPROVED** [review]

File: package-lock.json (hunk 2 above)
Comment (sorcerix) [reply package-lock.json:3931]: yeah, we just deleted it and then let it get created

File: components/TourBar/TourNavButton.tsx (hunk 1 above)
Comment (GrigoryGraborenko) [reply components/TourBar/TourNavButton.tsx]: fixed now

File: package-lock.json (hunk 2 above)
Comment (GrigoryGraborenko) [reply package-lock.json:3931]: No, but this is likely the result of failures to commit this file from earlier library additions. No NPM updates were run, only installs.

**Pull request merged**